src/org/zaproxy/zap/extension/webdriverLinux/ZapAddOn.xml

```diff
@@ -0,0 +1,25 @@
+<zapaddon>
+	<name>Linux Webdrivers</name>
```

Comment:
It could be WebDrivers instead, to be consistent with the docs in Selenium add-on (and the spec).